stable/prometheus-adapter/values.yaml

```diff
@@ -40,6 +40,7 @@ resources: {}
 
 rules:
   default: true
+  resources: false
```

Comment:
What occurs if the user simultaneously deploys both metrics-server and prometheus-adapter with this enabled? Is that supported or does it error?